user.css

```diff
@@ -264,7 +264,7 @@ html {
 .Root__top-bar {
   position: absolute !important;
   width: 100vw !important;
-  grid-area: none;
+  grid-area: main-view;
```

Comment:
and that completely breaks elements centering, also making properly centering impossible at all

Reply:
Main-view has been brought recently, by spotify, this makes the back and forward arrow buttons work

![image](https://user-images.githubusercontent.com/83197609/235176380-e4d7a4a7-103b-4480-93e9-94edac298457.png)
Without it

![image](https://user-images.githubusercontent.com/83197609/235176675-ac4da2ed-7fa9-4fd5-9c58-f66c8af3c44f.png)
With it

(unfortunately, I couldn't find a way to also include userwidget in the new UI)